frontend/components/panels/BaseClassPanel.js

```diff
@@ -39,9 +39,11 @@ class BaseClassPanel extends React.Component {
       renderedSections: this.props.aClass.sections.slice(0, sectionsShownByDefault),
       unrenderedSections: this.props.aClass.sections.slice(sectionsShownByDefault),
     };
+
+    this.onShowMoreClick = this.onShowMoreClick.bind(this);
   }
 
-  onShowMoreClick = () => {
```

Comment:
Lets change this back once babel sublime supports highlighting these as functions.